### DISCUSSION TITLE
What do we know about ion diffusivity in secondary particles which are agglomerates of primaries e.g. NMC

### DISCUSSION THREAD

**Discussion by TomTranter** [pybamm-team/PyBaMM#1286] [Q&A]
This paper https://onlinelibrary.wiley.com/doi/full/10.1002/aenm.202002655 shows nicely how cathode particles are agglomerates of smaller particles and depending on manufacturer can have very different shapes orientations grain boundaries and packing density. They also develop grain specific cracks. The primary morphology impacts the electrolyte accessibility and ion diffusivity but most models will assume a homogeneous diffusion coefficient. Could we do more to model these features and even aid their design?

**Comment by brosaplanella** [comment]
My guess (haven't done any calculations yet) is that if the particles are much smaller than the agglomerate (i.e. an agglomerate is made of many particles) you could use the homogenization theory to derive the effective diffusion coefficient and then you can treat them as a solid particle in the DFN. 

Here, [this paper by Maria Bruna and Jon Chapman](https://arxiv.org/pdf/1501.06549.pdf) might be (once again) very useful.

**Reply by TomTranter** [reply]
I'm sure you can but it begs the question what are the EIS measurements giving you. Diffusion in the electrolyte within the space between primaries or diffusion in the solid. This seems to be something particle manufacturers can tailor quite a bit and even make primaries of different shapes in different layers - see this paper for example https://pubs.rsc.org/no/content/articlehtml/2020/ee/d0ee02290j
I guess it's more a comment on what data we can trust in the literature? I wonder how much diffusivity in an NMC 811 particle can vary at fixed SOC based on primary particle morphology and packing? If it's an order or magnitude this could mess up some models if using a generic value...

**Reply by TomTranter** [reply]
Another consideration would be the effects of cracks (i.e. bigger openings between grain boundaries) which can span a significant part of the agglomerate. That might mess up the homogenisation but I don't know if it really matters. People are usually more concerned with loss of Li inventory or extra SEI when talking of cracks not necessarily the impact on diffusion.

**Reply by brosaplanella** [reply]
It would be interesting if that could be tested experimentally for different electrolytes (with different diffusivities) and also different particle packings. Then we would be able to tell if the homogenised results make sense.
